internal/cli/atlas/clusters/create.go

```diff
@@ -30,14 +30,13 @@ import (
 )
```

Comment:
```suggestion
// Copyright 2021 MongoDB Inc
```

Reply:
We don't need to change the year as we update files

Reply:
ah okay! Thanks!